packages/typescript/package.json

```diff
@@ -9,6 +9,7 @@
     "build-rules": "tsc"
   },
   "devDependencies": {
+    "typescript": "^3.0.0",
```

Comment:
Maybe we could start with everyone fixed to the `3.8.3`

Reply:
that would reduce the number of parallel installed typescript packages and would lower the amount of installed npm packages in general 👍